templates/configmaps/chain-coord-deployment-plan.template.yaml

```diff
@@ -3,5 +3,5 @@ data:
   key: "{value}"
 kind: ConfigMap
 metadata:
-  name: stacks-api-conf
+  name: deployment-plan
```

Comment:
The way git is reconciling the deployment-plan CM is confusing, but I think everything is ok here.

Reply:
Yeah :/

I thought it was the way vscode/git were auto-`git mv`-ing when I renamed/moved the files. I noticed the oddity in #65.

So I just started from a clean slate and did a manual `git mv ./templates/name-asset.template.yaml ./templates/assets/name.template.yaml` for all pods/configmaps/services, and it still is getting mixed up. So I don't know what's going on here.

It seems the actual contents are correct, just the git histories for the files are mixed up.